src/components/Dropdown/Items/SingleSelectItem/SingleSelectItem.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { DropdownMenuItem, DropdownMenuItemProps } from '@radix-ui/react-dropdown-menu';
+import { bem } from '../../../../utils';
+import styles from '../Item.scss';
+
+export interface Props extends Omit<DropdownMenuItemProps, 'onSelect'> {
+    /** A function to be called if the item is clicked */
+    onSelect?: (e: React.SyntheticEvent<HTMLDivElement>) => void;
+    /** Id for the checkbox */
+    id?: string;
+    /** to add a priority badge before the label */
+    hasPriority?: boolean;
+    /** Checkbox status */
+    isSelected?: boolean;
+}
+
+const { block } = bem('DropdownItem', styles);
+
+export const SingleSelectItem = React.forwardRef<HTMLDivElement, Props>(
```

Comment:
the cursor of any item (single, multi) looks a little bit confusing:
https://github.com/user-attachments/assets/9919dcf0-1cf5-48bb-96c1-faf16fb9aa83